docs/topic-guides/elastic-infra/index.txt

```diff
@@ -1,51 +1,48 @@
 .. _elastic-infra-index:
 
 Elastic Infrastructure
-=============================
+======================
 
-This document describes how Determined automatically scales its cluster to
-provide adequate computation resources for workloads as well as leverage
-computational resources. It assumes some familiarity with the terms make
-in :ref:`det-system-architecture`: **master**, **agent**, **slot**, and
-**workload**.
+When running in a cloud environment, Determined can automatically
```

Comment:
non-blocking: "Determined *will* automatically" as a completely optional suggestion to sound more confident :)

Reply:
or even: "Determined automatically provisions and terminates..."

Reply:
You can deploy Determined in the cloud using only static agents, in which case it won't provision or terminate instances automatically. That isn't a super common configuration but I can imagine some scenarios where it might be useful.